compiler/damlc/tests/daml-test-files/Math.daml

```diff
@@ -45,6 +45,14 @@ powerTest = scenario do
   let
     sqrt2 = 1.4142135623 : Decimal
   assert(abs(sqrt2 - 2.0 ** 0.5) <= 0.0000000001)
+  assert(abs(0 ** 0) == 1.0)
```

Comment:
```suggestion
 0 ** 0 === 1.0
```
1. no reason for abs
2. === shows you the values that didn’t match whereas `assert` just tells you that something failed which isn’t all that helpful.